languageserver/server/server.go

```diff
@@ -1963,6 +1984,8 @@ func (s *Server) getDiagnostics(
 		diagnostics = append(diagnostics, diagnostic)
 	}
 
+	analysisProgram.Run(maps.Values(linter.Analyzers), report)
```

Comment:
Might be good to "cache" the result of `maps.Values(linter.Analyzers)`, maybe even on the linter side, e.g. through a getter function that memoizes